.github/workflows/ci_tests.yaml

```diff
@@ -123,7 +123,7 @@ jobs:
       # Install build dependencies from conda-forge
       - name: Install build dependencies
         shell: bash -l {0}
-        run: conda install cmake
+        run: conda install cmake netcdf
```

Comment:
@weiji14 It should be `libnetcdf`. 

[This file](https://github.com/conda-forge/gmt-feedstock/blob/master/recipe/meta.yaml#L22) may be helpful.